app/scripts/lib/mailcheck.js

```diff
@@ -8,20 +8,17 @@
 /* exceptsPaths: mailcheck */
 define([
   'views/tooltip',
-  'lib/url',
   'mailcheck'
 ],
-function (Tooltip, Url) {
+function (Tooltip) {
 
-  var DOMAINS = [ // domains that get suggested, i.e gnail.com => gmail.com
-    'gmail.com', 'qq.com', 'yandex.ru', 'o2.pl', 'rambler.ru', 'googlemail.com', 't-online.de', 'mail.ru', 'web.de',
-    'sbcglobal.net', 'msn.com', 'me.com', 'aol.de', 'aol.com', 'seznam.cz', 'comcast.net', 'orange.fr',
-    'gmx.de', 'ymail.com', 'outlook.com', 'live.com', 'yahoo.com', 'yahoo.fr', 'yahoo.co.id'
-  ];
-  var SECOND_LEVEL_DOMAINS = [];
-  var TOP_LEVEL_DOMAINS = [ // TLD suggestion, i.e some.cpm => some.com
-    'com', 'net', 'org', 'ru', 'jp', 'de', 'fr', 'pl', 'es', 'co.uk', 'bg', 'co.id', 'cz'
+  var DOMAINS = [];
+  var SECOND_LEVEL_DOMAINS = [ // domains that get suggested, i.e gnail => gmail
+    'gmail', 'qq', 'yandex', 'o2', 'rambler', 'googlemail', 't-online', 'mail', 'web',
```

Comment:
With the change from DOMAIN to SECOND_LEVEL_DOMAINS, will potentially erroneous addresses like `gmail.con` still show the suggestion?